server/src/modules/users/users.repo.ts

```diff
@@ -21,28 +21,36 @@ export class UsersRepo {
      * @summary Gets all from database
      * @returns All users
      */
-    async GetAll(where?: Prisma.UserWhereInput, skip?: number, take?: number): Promise<[User[], number]> {
+    async GetAll(
+        where: Prisma.UserWhereInput,
+        include: Prisma.UserInclude,
+        skip?: number,
+        take?: number
+    ): Promise<[User[], number]> {
         const count = await this.prisma.user.count({
             where: where
         });
+
         const users = await this.prisma.user.findMany({
             where: where,
-            skip: skip != null ? +skip : undefined,
-            take: take != null ? +take : undefined
+            include: include,
+            skip: skip,
```

Comment:
This with its removed `+` (e.g `skip: +skip`) still still causes a Prisma runtime error

Reply:
Example error:
```
Invalid `this.prisma.user.findMany()` invocation in
F:\Users\alexd\Repos\GitHub\Others\momentum-mod\website-nestJS\server\src\modules\users\users.repo.ts:34:46

  31     where: where
  32 });
  33
→ 34 const users = await this.prisma.user.findMany({
       where: {},
       include: {
         profile: undefined,
         userStats: undefined
       },
       skip: '0',
             ~~~
       take: '20'
             ~~~~
     })

Argument skip: Got invalid value '0' on prisma.findManyUser. Provided String, expected Int.
Argument take: Got invalid value '20' on prisma.findManyUser. Provided String, expected Int.
```

Reply:
Huh, is this coming from a test or running manually? Never seen this myself, ParseIntPipe should be handling it

Reply:
I ran this manually but tests would have caught it too, I just didn't use them yet